src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -6,9 +6,15 @@
 public class CliSyntax {
 
     /* Prefix definitions */
+    public static final Prefix PREFIX_COMPANY = new Prefix("c/");
+    public static final Prefix PREFIX_CONTACT = new Prefix("p/");
```

Comment:
`PREFIX_CONTACT` and `PREFIX_POSTION` are set to be the same `"p/"` here.
Suggesting to change the `PREFIX_CONTACT` to `"cn/"`, short for `contact number`.

```suggestion
    public static final Prefix PREFIX_CONTACT = new Prefix("cn/");
```

Reply:
fixed